check_tags_in_requests.py

```diff
@@ -60,18 +60,56 @@ def __init__(self, *args, **kwargs):
         if self.factory is None:
             self.factory = "openSUSE:Factory"
         super(TagChecker, self).__init__(*args, **kwargs)
-        needed_tags = [r'bnc#[0-9]+',
+        # try to keep in sync (same sorting ) with
+        # https://en.opensuse.org/openSUSE:Packaging_Patches_guidelines#Current_set_of_abbreviations
+        needed_tags = [r'boo#[0-9]+',
+                       r'pr#[0-9]+',
+                       r'boost#[0-9]+',
+                       r'ceph#[0-9]+',
+                       r'claws#[0-9]+',
                        r'cve-[0-9]{4}-[0-9]+',
```

Comment:
Nitpicking: the minimal number of digits for the second part of a CVE identifier is four (seehttps://cve.mitre.org/cve/identifiers/syntaxchange.html#new), it should be actually `cve-[0-9]{4}-[0-9]{4,}`...